pqt.so/include/action.h

```diff
@@ -48,6 +48,11 @@ using namespace std::chrono;
 
 class pqtworker;
 
+typedef enum class pqt_json_data {

```

Comment:
I dont think a typedef is needed. Enum class defines a type.

Reply:
Yes, it's implicit. Removed typedef.